waku/waku_test.go

```diff
@@ -1061,15 +1091,27 @@ func TestHandleP2PMessageCode(t *testing.T) {
 	InitSingleTest()
 
 	w1 := New(nil, nil)
-	w1.SetMinimumPoW(0.0000001, false) // nolint: errcheck
-	w1.Start(nil)                      // nolint: errcheck
+	if err := w1.SetMinimumPoW(0.0000001, false); err != nil {
+		t.Error(err)
+	}
+	if err := w1.Start(nil); err != nil {
+		t.Error(err)
+	}
 
-	defer w1.Stop() // nolint: errcheck
+	defer func() {
+		handleError(t, w1.Stop())
+	}()
 
 	w2 := New(nil, nil)
-	w2.SetMinimumPoW(0.0000001, false) // nolint: errcheck
-	w2.Start(nil)                      // nolint: errcheck
-	defer w2.Stop()                    // nolint: errcheck
+	if err := w2.SetMinimumPoW(0.0000001, false); err != nil {
+		t.Error(err)
+	}
+	if err := w2.Start(nil); err != nil {
+		t.Error(err)
+	}
+	defer func() {
+		handleError(t, w2.Stop())
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)